ext/README.md

```diff
@@ -0,0 +1,49 @@
+# Extensions
+
+The basis of the Extension API is described in a document which should be emulated for all extensions.
+
+## Table of Contents
+
+<!-- toc -->
+* [Table](#table)
+* [Name](#name)
+* [Filename](#filename)
+* [Detail](#detail)
+* [Prior Art](#prior-art)
+<!-- /toc -->
+
+## Table
+
+_notice_: All new `./ext/ext-$name.md` docs MUST be added to this table.
+
+| `$name`  | Definition | Summary |
+|:--:|:--:|:--:|
+| 0 | [ext-0](./ext-0.md) | Base definition of discovering extensions on registry server |
+|  |  |  |
+
+
+## Name
+
+Extension names MUST be unique.
+Names SHOULD include a version.
```

Comment:
What model for version naming should be used

Reply:
yes?
I looked at the kube CRD versioning, but didn't care to pull that in just yet.